govuk_chat_evaluation/rag_answers/custom_deepeval/metrics/absence_of_factual_contradictions/schema.py

```diff
@@ -0,0 +1,33 @@
+from typing import Literal
+from pydantic import BaseModel, Field
+
+
+class Verdict(BaseModel):
+    verdict: Literal["yes", "no", "i do not know"]
+    reason: str | None = Field(default=None)
```

Comment:
```suggestion
    reason: str | None = None
```

minor: I think this only needs this rather than a Field